config/locales/views/reactions/en.yml

```diff
@@ -14,6 +14,8 @@ en:
         lightbulb: Lightbulb
         raised_hands: Raised hands
         rocket: Rocket
+        exploding-head: Exploding head
```

Comment:
This is maybe unnecessary (and certainly will be in the long term) but: if we toggle the feature flag on and back off again (a rollback scenario), reactions in the notifications area will throw translation exceptions. (I _think_ those exceptions are handled more smoothly in production, but I'm not confident of that.)